### PR TITLE
[Twitter Adapter] Add unit tests for WebhooksEnterpriseManager class

### PR DESCRIPTION
**Pull request by sw-joelmut**
## Description
This PR adds unit tests for the [WebhooksEnterpriseManager](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet/blob/develop/libraries/Bot.Builder.Community.Adapters.Twitter/Webhooks/Services/WebhooksEnterpriseManager.cs#L17) class increasing its code coverage from **14%** to **79%**.

### Specific Changes
- Updated the [WebhooksEnterpriseManagerTests](https://github.com/BotBuilderCommunity/botbuilder-community-dotnet/blob/develop/tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs#L12) file with new unit tests.
- Migrated existing tests to xUnit.

## Testing
The following images show the new tests passing and the resulting code coverage.
image
image

### CODE REVIEW

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs [hunk 1]
```diff
@@ -13,12 +13,59 @@ public class WebhooksEnterpriseManagerTests
...
+
+            Assert.AreEqual(89, result.Error.Errors[0].Code);
+        }
+
+        [TestMethod]
+        public async Task RegisterWebhookShouldReturnUnauthorized()
+        {
+            var enterpriseManager = new WebhooksEnterpriseManager(_testOptions.Object.Value);
+
+            var result = await enterpriseManager.RegisterWebhook("3");
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs]: Can we use a better fake URL here?

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs [hunk 2]
```diff
@@ -13,12 +13,59 @@ public class WebhooksEnterpriseManagerTests
...
 
             await Assert.ThrowsExceptionAsync<ArgumentException>(async () => { await enterpriseManager.RegisterWebhook(string.Empty); });
         }
+
+        [TestMethod]
+        public async Task UnregisterWebhookShouldReturnUnauthorized()
+        {
+            var enterpriseManager = new WebhooksEnterpriseManager(_testOptions.Object.Value);
+
+            var result = await enterpriseManager.UnregisterWebhook("3");
```
Comment (ceciliaavila) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs]: The same in here.

**Review by ceciliaavila — CHANGES_REQUESTED** [review]

**Review by ceciliaavila — APPROVED** [review]

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs [hunk 3]
```diff
@@ -1,24 +1,72 @@
...
+            };
+
+            _testOptions.SetupGet(x => x.Value).Returns(options);
+        }
+
+        [Fact]
+        public async Task GetRegisteredWebhooksShouldReturnUnauthorized()
+        {
+            var enterpriseManager = new WebhooksEnterpriseManager(_testOptions.Object.Value);
+
```
Comment (santgr11) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs]: remove the space between var declarations

File: tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs [hunk 4]
```diff
@@ -1,24 +1,72 @@
...
 
-            await Assert.ThrowsExceptionAsync<ArgumentException>(async () => { await enterpriseManager.RegisterWebhook(string.Empty); });
+            await Assert.ThrowsAsync<ArgumentException>(async () => { await enterpriseManager.RegisterWebhook(string.Empty); });
+        }
+
+        [Fact]
+        public async Task UnregisterWebhookShouldReturnUnauthorized()
+        {
+            var enterpriseManager = new WebhooksEnterpriseManager(_testOptions.Object.Value);
+
```
Comment (santgr11) [inline tests/Bot.Builder.Community.Adapters.Tests/Webhooks/Services/WebhooksEnterpriseManagerTests.cs]: same here

**Review by santgr11 — CHANGES_REQUESTED** [review]

**Review by ian-whiteout — APPROVED** [review]

**Pull request closed without merging**